### PR TITLE
Add example projects page

### PR DESCRIPTION
**Pull request by jettify**
## What do these changes do?
As discussed in https://github.com/aio-libs/aiohttp-demos/issues/32 I am adding page with list of example projects to Misc subsection.

## Related issue number
https://github.com/aio-libs/aiohttp-demos/issues/32

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- `imagetagger <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/imagetagger>`_
+  Example how to deploy deep learning model with aiohttp.
+
+- `chat <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/chat>`_
+  Simple chat using websockets.
+
+- `polls <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/chat>`_
+  Simple *polls* application with PostgreSQL storage.
+
+- `Blog <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/chat>`_
```
Comment (webknjaz) [inline docs/examples.rst]: This thing is titlecased, which others ain't. Maybe it's worth keeping the same style

Comment (jettify) [reply docs/examples.rst]: done

File: docs/examples.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
+.. _aiohttp-examples:
+
+================
+Example Projects
```
Comment (webknjaz) [inline docs/examples.rst:4]: Demo projects maybe?

**Comment by asvetlov** [comment]
I think better to keep the list in aiohttp-demos documentation and add a link to demos page into aiohttp docs.

**Comment by jettify** [comment]
Ok, not sure about the place where to put it, we already have a link in README.

File: docs/examples.rst (hunk 2 above)
Comment (Arfey) [reply docs/examples.rst:4]: why?

File: docs/examples.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- `motortwit <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/motortwit>`_
+  Twitter clone with MongoDB storage.
+
+- `imagetagger <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/imagetagger>`_
+  Example how to deploy deep learning model with aiohttp.
+
+- `chat <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/chat>`_
+  Simple chat using websockets.
+
+- `polls <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/chat>`_
```
Comment (Arfey) [inline docs/examples.rst:34]: replace link to `https://github.com/aio-libs/aiohttp-demos/tree/master/demos/polls`

File: docs/examples.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  Simple *polls* application with PostgreSQL storage.
+
+- `blog <https://github.com/aio-libs/aiohttp-demos/tree/master/demos/chat>`_
```
Comment (Arfey) [inline docs/examples.rst:37]: `https://github.com/aio-libs/aiohttp-demos/tree/master/demos/blog`

**Comment by asvetlov** [comment]
What's about https://demos.aiohttp.org/en/latest/ ?

**Comment by Arfey** [comment]
@asvetlov 10 min 😀

**Pull request closed without merging**